tests/debugger/do_backtrace.expected

```diff
@@ -14,10 +14,15 @@ out: function test
 Stopped at tests/debugger/do_backtrace.js:33 (in test() at line:30, col:1)
 (jerry-debugger) s
 Stopped at tests/debugger/do_backtrace.js:23 (in foo() at line:21, col:1)
+(jerry-debugger) bt 1 2
+Frame 0: tests/debugger/do_backtrace.js:23 (in foo() at line:21, col:1)
+Frame 1: tests/debugger/do_backtrace.js:33 (in test() at line:30, col:1)
 (jerry-debugger) bt
 Frame 0: tests/debugger/do_backtrace.js:23 (in foo() at line:21, col:1)
 Frame 1: tests/debugger/do_backtrace.js:33 (in test() at line:30, col:1)
 Frame 2: tests/debugger/do_backtrace.js:40
+(jerry-debugger) bt 1 1
+Frame 0: tests/debugger/do_backtrace.js:23 (in foo() at line:21, col:1)
```

Comment:
When you specify a min, then maximum (max-min) items could be sent. Here (1-1)==0, so the backtrace should be empty (empty backtrace is valid, you need to test it!!!)

So "bt 1 2" above should only send 1 item. Please also test a large number, e.g. "bt 1000 1001".